apache-spark/spark3.0/image-classification/src/main/scala/com/examples/SparkModel.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.examples
+
+import ai.djl.inference.Predictor
+import ai.djl.modality.Classifications
+import ai.djl.repository.zoo.{Criteria, ModelZoo}
+import ai.djl.training.util.ProgressBar
+import org.apache.spark.sql.Row
+
+@SerialVersionUID(123456789L)
+class SparkModel(val url : String) extends Serializable {
+
+  private lazy val criteria = Criteria.builder
+    .setTypes(classOf[Row], classOf[Classifications])
+    .optModelUrls(url)
+    .optTranslator(new SparkImageClassificationTranslator())
```

Comment:
is this translator serializable? If not, how do we make sure SparkImageClassificationTranslator being used in each executor?

Reply:
Yes it should be serializable